docs/api/README.md

```diff
@@ -0,0 +1,190 @@
+# Flipper::Api
+
+API for the [Flipper](https://github.com/jnunemaker/flipper) gem.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+    gem 'flipper-api'
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install flipper-api
+
+## Endpoints
+
+### Get all features
+
+`GET /flipper-api/api/v1/features`
```

Comment:
One thing I love to do when documenting API's is include a curl command. For example, here is what I did for Gauges way back in the day: http://get.gaug.es/documentation/reference-listing/your-information/.

![gaugesdocs](https://cloud.githubusercontent.com/assets/235/19280132/6e40dc9a-8fb2-11e6-825a-29a1cf1bbbc5.png)

What do you think about that? Path, Request (as curl) and Response? Does that seem like it would work here or does it spur any other ideas?